controllers/user.js

```diff
@@ -52,13 +56,15 @@ const login = async (req, res) => {
 }
 
 const logout = async (req, res) => {
-  try {
-    res.clearCookie('userToken')
-    res.sendStatus(200)
-  } catch (err) {
-    console.log(err)
-    res.sendStatus(500)
-  }
+  console.log(req.body)
+  // try {
+  res.clearCookie('userToken')
+  res.redirect('/login')
```

Comment:
If it helps, perhaps you can first `await` the cookie to be cleared, then redirect the user